src/buffer/out/UTextAdapter.cpp

```diff
@@ -417,10 +417,7 @@ til::point_span Microsoft::Console::ICU::BufferRangeFromMatch(UText* ut, URegula
 {
     UErrorCode status = U_ZERO_ERROR;
     const auto nativeIndexBeg = uregex_start64(re, 0, &status);
-    auto nativeIndexEnd = uregex_end64(re, 0, &status);
-
-    // The parameters are given as a half-open [beg,end) range, but the point_span we return in closed [beg,end].

```

Comment:
the comment says it returns an inclusive range - is that no longer true?

Reply:
Correct, it's a half-open range now. Updated the comment.